atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/schema/KeyValueServiceValidator.java

```diff
@@ -100,6 +107,13 @@ private void validateTables(Set<TableReference> tables) {
             Future<Void> future = executor.submit(() -> {
                 try {
                     validateTable(table);
+                    KeyValueServiceMigratorUtils.processMessage(
```

Comment:
non-actionable: I'll admit I don't quite see the benefit of what using `KeyValueServiceMigratorUtils#processMessage` provides over just calling the messageProcessor ourselves, but it's been that way before and so this is consistent.

Reply:
Yeah I opted not to modify this given limited bandwidth.